config_defaults/subtests/docker_cli/negativeusage.ini

```diff
@@ -1,5 +1,5 @@
 [docker_cli/negativeusage]
-subsubtests = op1, op2, op3, if1, ov1, iv1, iv3, iv4, iv5, iv6, iv7, ip1
+subsubtests = op1, op2, op3, if1, ov1, iv1, iv3, iv4, ip1
```

Comment:
Hrmmm, oops, totally wrong.  There's nothing special done for this test in CI.  Wonder why it failed on mismatch w/ the ``subsubtests`` list...